### PR TITLE
management command to import state information field types and data from prod

### PR DESCRIPTION
**Pull request by jlev**
[ch402]
adds serializer field to check format. will not work until that's deployed

### CODE REVIEW

File: app/election/management/commands/importfromprod.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        parser.add_argument(
+            "--api-url",
+            nargs=1,
+            type=str,
+            default="https://api.voteamerica.us/v1",
+            help="API to import from",
+        )
+
+    def handle(self, *args, **options):
+        API_URL = options["api_url"]
```
[the remark this answers is not in the document]
Comment (jlev) [reply app/election/management/commands/importfromprod.py]: weird that the argparse options are a list, when the default was a string. fixed

File: app/election/management/commands/importfromprod.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            state = State.objects.get(code=state_code)
+            if not state:
+                self.stdout.write(f"Not Found {state_code}")
+                continue
+
+            for state_information in obj.get('state_information'):
+                info, created = StateInformation.objects.get_or_create(
+                    state=state,
+                    field_type__slug=state_information.get('field_type')
+                )
```
[the remark this answers is not in the document]
Comment (jlev) [reply app/election/management/commands/importfromprod.py]: Fair. I was trying to be explicit, but this is indeed much faster. Forgot that the StateInformation manager handles select_related and StateInformationFieldType pre-created all the objects. No more get_or_create.

**Comment by jlev** [comment]
Need to push to staging or dev to test.

**Comment by nickcatal** [comment]
@jlev `make shelldev` should get you there

**Comment by jlev** [comment]
tested as working with `python manage.py importfromprod --api-url=https://api-staging.voteamerica.us/v1`

**Pull request merged**